python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -364,6 +372,32 @@ def run(
 
         return subtractorExp, invImages, badStamps
 
+    def detInRegions(self, inputs):
+        """Determine whether the input exposure's detector is in the region(s)
+        where the extended PSF model(s) is(are) available.
+
+        Parameters
+        ----------
+        inputs : `dict`
+            Dictionary containing the inputs to the task, including
+            `inputExposure` and `inputExtendedPsf`.
```

Comment:
Rather than a single global `inputs` dict here, I think it would make more sense to split this into two required inputs: `inputExposure` and `inputExtendedPsf`. This will make it clear exactly what inputs from the input dict are required, what their purposes are, and prevent accidental usage of any other input.